functions/common.py

```diff
@@ -28,7 +28,7 @@ def factorial(argument: int) -> int:
         result *= i
     return result
 
-
+# Note - I don't think this is actually used anywhere
```

Comment:
It isn't. I removed it at one point but it got added back so I added testing for it in my last PR. I remember having issues with it actually but I can't remember what they were anymore.